examples/getting-started-movielens/01-Download-Convert.ipynb

```diff
@@ -212,13 +215,13 @@
        "4                                       Comedy  "
       ]
      },
-     "execution_count": 6,
+     "execution_count": 8,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "movies = pd.read_csv(BASE_DIR + 'ml-25m/movies.csv')\n",
+    "movies = cudf.read_csv(os.path.join(BASE_DIR, 'ml-25m/movies.csv'))\n",
```

Comment:
thanks for this! Can we also remove the `import pandas as pd` from the top of the file? its no longer used

Reply:
Sure. Done